starter-core/src/main/resources/pom.xml

```diff
@@ -157,5 +157,21 @@
             <artifactId>plugin</artifactId>
             <version>5.12.0</version>
         </dependency>
+        <dependency>
+            <!-- FIXME: this can be removed once the micronaut-bom references version 3.1.2 or higher -->
+            <groupId>io.micronaut.views</groupId>
+            <artifactId>micronaut-views-jte</artifactId>
+            <version>3.1.2</version>
```

Comment:
This can be removed

Reply:
My build is currently pulling micronaut-bom-3.2.8-SNAPSHOT and this still refers to micronaut-views-bom:3.0.4, which doesn't include micronaut-views-jte.

Is there something else I need to do to pick up the newer version of micronaut-views?